src/bower/DependenciesManager.js

```diff
@@ -77,10 +77,41 @@ define(function (require, exports) {
         return FileUtils.exists(path);
     }
 
+    /**
+     * Get the dir of the dependencies
+     * @param {string=} path
+     * @return {Promise}
+     */
+    function findBowerJsonLocation(path) {
+        if (!path) {
```

Comment:
This one returns the directory of bower.json.
I modified the comment now. will commit when all is ok